libs/providers/ofrep-web/src/lib/model/ofrep-web-provider-options.ts

```diff
@@ -0,0 +1,15 @@
+export interface OfrepWebProviderOptions {
```

Comment:
minor - we should expose a mechanism to set authN headers. See comment [1] for server provider 

[1] - https://github.com/open-feature/js-sdk-contrib/pull/817#discussion_r1546648202